API.md

```diff
@@ -2310,7 +2310,7 @@ following options:
           clean up the files generated by the framework. This can be done by keeping track
           of which files are used (e.g. using the `request.app` object), and listening to
           the server `'response'` event to perform any needed cleanup.
-    - `parse` - can be `true`, `false`, or `gunzip`; determines if the incoming payload is
+    - `parse` - can be `true`, `false`, `gunzip`, or `br`; determines if the incoming payload is
```

Comment:
fair enough, silly decision on their part IMO, but if it's an established thing then we should probably stick with it